pkg/reconciler/sinkbinding/controller.go

```diff
@@ -114,6 +135,20 @@ func NewController(
 		},
 	}
 
+	// Reconcile SinkBinding when the OIDC service account changes
+	serviceaccountInformer.Informer().AddEventHandler(cache.FilteringResourceEventHandler{
+		FilterFunc: controller.FilterController(&v1.SinkBinding{}),
+		Handler:    controller.HandleAll(impl.EnqueueControllerOf),
+	})
+
```

Comment:
@creydr it's a general common not specific to this PR, but we should evaluate if it makes sense to label all of our service accounts and use a "filtered informer" so that we're not watching **all** service accounts into a cluster which could be quite large but only "ours"

Reply:
I think it makes sense to label the service account 👍🏼

Reply:
That's a good point @pierDipi. I think we can do this in a separate PR

Reply:
created #7341